lighthouse-core/audits/time-to-interactive.js

```diff
@@ -27,8 +27,8 @@ class TTIMetric extends Audit {
     return {
       category: 'Performance',
       name: 'time-to-interactive',
-      description: 'Time To Interactive (alpha)',
```

Comment:
nope.  :)
 that's https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/first-interactive.js which is a different metric altogether.